cloudfour/cloudfour.com-patterns
# Text inputs

## Overview

Introduces a `c-input` component that covers text-based inputs. It works in default and dark themes and has `readonly`, `disabled`, `focus` and `hover` styles.

Also normalizes heights and dimensions between inputs and buttons so they should look nice in proximity regardless of type or browser.

It also adds a downward caret icon intended for `select` support, but I started going down a rabbit hole of SVG inlining and decided to pause and submit a PR.

## Screenshots

<img width="1102" alt="Screen Shot 2020-04-01 at 3 27 32 PM" src="https://user-images.githubusercontent.com/69633/78192767-ac392d00-742d-11ea-8846-83703589eb08.png">

## Testing

1. View Component/Input story in deploy preview.
1. Try various knobs and confirm that they work as expected.
1. Try various knobs again after switching to the Dark theme from the Storybook toolbar.

---

- See #490
